src/app/workflow/versions/versions.component.ts

```diff
@@ -82,6 +82,9 @@ export class VersionsWorkflowComponent extends Versions implements OnInit, OnCha
   @Input() workflowId: number;
   @Input() verifiedVersionPlatforms: Array<VersionVerifiedPlatform>;
   @Input() publicPage: boolean;
+  @Input() hasExecutionMetrics!: boolean;
+  @Input() hasValidationMetrics!: boolean;
```

Comment:
What does the exclamation point do?

Reply:
Weird, I think Intellij added that on its own when I created the inputs through their dialog